include/exadg/convection_diffusion/time_integration/time_int_bdf.cpp

```diff
@@ -68,16 +68,17 @@ TimeIntBDF<dim, Number>::setup_derived()
   {
     // compute the grid coordinates at start time (and at previous times in case of
     // start_with_low_order == false)
-
-    pde_operator->move_grid(this->get_time());
+    pde_operator->move_grid(this->get_time(),
+                            false /* only update preconditioner prior to ALE update */);
```

Comment:
I have problems understanding this comment, probably the notion "ALE update" is not fully clear to me at this point.